cmd/exo/convert.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/deref/exo/internal/manifest"
+	"github.com/deref/exo/internal/manifest/compose"
+	"github.com/deref/exo/internal/manifest/procfile"
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(convertCmd)
+	convertCmd.Flags().StringVar(&convertFlags.Format, "format", "", "compose, procfile")
+}
+
+var convertFlags struct {
+	Format string
+}
+
+var convertCmd = &cobra.Command{
+	Use:   "convert [flags] [manifest-file]",
+	Short: "Converts a docker-compose file or Procfile into an exo manifest",
+	Long: `If unspecified, a manifest format will be guessed from the manifest filename.  This can be
```

Comment:
iirc, the short message doesn't appear at all when you do `exo help convert`, and so you should more-or-less duplicate the short string in to the long string as well.